indigo_app/views.py

```diff
@@ -269,17 +268,15 @@ def post(self, request, frbr_uri, version_id):
         return redirect(url)
 
 
-class BatchAddWorkView(AbstractAuthedIndigoView, TemplateView):
+class BatchAddWorkView(AbstractAuthedIndigoView, FormView):
     template_name = 'work/new_batch.html'
     # permissions
     permission_required = ('indigo_api.add_work',)
+    form_class = BatchCreateWorkForm
 
-    def get_context_data(self, **kwargs):
-        context = super(BatchAddWorkView, self).get_context_data(**kwargs)
+    def form_valid(self, form):
```

Comment:
We're only going to work with the form if the basic validation in place succeeds.